Mathlib/Analysis/SpecialFunctions/Trigonometric/Sinc.lean

```diff
@@ -0,0 +1,89 @@
+/-
+Copyright (c) 2025 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import Mathlib.Analysis.SpecialFunctions.Trigonometric.Bounds
+import Mathlib.Analysis.Calculus.DSlope
+
+/-!
+# Sinc function
+
+This file contains the definition of the sinc function and some of its properties.
+
+## Main definitions
+
+* `Real.sinc`: the sinc function, defined as `sinc x = sin x / x` for `x ≠ 0` and `1` for `x = 0`.
```

Comment:
```suggestion
* `Real.sinc`: the (unnormalized) sinc function, defined as `sinc x = sin x / x` for `x ≠ 0` and `1` for `x = 0`.
```

since https://en.wikipedia.org/wiki/Sinc_function suggests that different fields use a different scale factor. Perhaps worth adding the link to the references, even if its obvious.